engine/executor/server.go

```diff
@@ -193,6 +201,12 @@ func (s *Server) makeTask(
 		log.L().Error("Failed to create worker", zap.Error(err))
 		return nil, err
 	}
+	if jm, ok := newWorker.(lib.BaseJobMaster); ok {
+		engine := gin.New()
+		apiGroup := engine.Group(jobAPIBasePath(jm.JobMasterID()))
+		jm.OnOpenAPIInitialized(apiGroup)
```

Comment:
`OnOpenAPIInitialized` is called before `Init`. Could you add some comments?

Reply:
I will add it later.